frontends/ol-components/src/components/Button/Button.tsx

```diff
@@ -170,10 +170,12 @@ const ButtonInner: React.FC<
   )
 }
 
-const Button: React.FC<ButtonProps> = ({ children, ...props }) => (
-  <ButtonStyled type="button" {...props}>
-    <ButtonInner {...props}>{children}</ButtonInner>
-  </ButtonStyled>
+const Button = React.forwardRef<HTMLButtonElement, ButtonProps>(
```

Comment:
Incidentally, `forwardRef` is getting deprecated in React 19: https://react.dev/blog/2024/04/25/react-19#ref-as-a-prop yaaaaaaay